test/index.js

```diff
@@ -1,9 +1,7 @@
-var path = require('path')
+//
 var tape = require('tape')
 
-var test = require('../')(tape, {
-  fixtures: path.join(__dirname, 'fixtures')
-})
+var test = require('../')(tape)
```

Comment:
I'd prefer two have two tests: one with no options and one with options that have "fixtures" in a different location.